src/sanitizer.js

```diff
@@ -186,6 +190,16 @@ export function sanitizeHtml(html) {
         }
         return;
       }
+      const bindAttribsIndices = {};
+      // Special handling for attributes for amp-bind which are formatted as
+      // [attr]. The brackets are restored at the end of this function.
+      for (let i = 0; i < attribs.length; i += 2) {
+        const attr = attribs[i];
+        if (attr && attr[0] == '[' && attr[attr.length - 1] == ']') {
+          bindAttribsIndices[i] = true;
+          attribs[i] = attr.slice(1, -1);
+        }
+      }
       if (BLACKLISTED_TAGS[tagName]) {
         ignore++;
       } else if (tagName.indexOf('amp-') != 0) {
```

Comment:
Changing it will change the history for the line, misleading anyone who looks in the future to believe that the change is related to supporting mustache templates.